src/raftstore/store/msg.rs

```diff
@@ -32,6 +32,7 @@ pub enum Tick {
     RaftLogGc,
     SplitRegionCheck,
     PdHeartbeat,
```

Comment:
What's the difference between PdHeartbeat and PdRegionHeartbeat ?  Should we rename PdHeartbeat ?